Sources/XCHammer/XCBuildSettings.swift

```diff
@@ -163,6 +166,11 @@ struct XCBuildSettings: Encodable {
     var useHeaderMap: First<String>? = First("NO")
     var testTargetName: First<String>?
     var pythonPath: First<String>?
+    var sdkRoot: First<String>?
+    var targetedDeviceFamily: OrderedArray<String> = OrderedArray.empty
```

Comment:
What's an `OrderedArray`?

Reply:
Ordered array is poorly named, it's basically an implementation of ordered set that conform to a few protocols used to serialize build settings. We wanted uniqueness with consistent ordering which is why this was originally written, happy to revisit